### DISCUSSION TITLE
Single guest session issue

### DISCUSSION THREAD

**Discussion by wolfclan68** [ltsp/ltsp#204] [Q&A]
@enoch85 I followed your guide and double-checked everything, but I'm still getting guest account logged in and appearing like a new user with the "What's new in Ubuntu" splash. I'm not sure what I'm missing. I did not do the Rdesktop section, but it did not look like that should be related. Of course, I'm probably wrong about that.

**Comment by wolfclan68** [comment]
So I went through and tried to check what was or was not happening. It seems the following commands are either not running or being denied, not sure how to check which. 
mount 192.168.2.4:/home/nfs /mnt
mkdir -p /mnt/$MAC_ADDRESS/home/guest
chown guest:guest /mnt/$MAC_ADDRESS/home/guest

The /home/nfs has MAC address file after a client boot, but I see nothing in the /mnt directory and the /etc/guest-template is not being generated.

Any help is appreciated.

**Comment by alkisg** [comment]
Put this is ltsp.conf: `POST_INIT_ZZZ_DEBUG="debug_shell"`

Run `ltsp initrd`, reboot the client, it'll end up to a shell right after all the POST_INIT commands were executed. Get a photo with your phone and upload it here, so that we can see if there were any errors, warnings etc. Then type Ctrl+D to continue booting. After the client is booted, run the following commands and paste their outputs:

```shell
showmount -e 192.168.2.4
grep -i nfs /proc/self/mountinfo
```

**Comment by enoch85** [comment]
@wolfclan68 Please note that 192.168.2.4 is my IP of the server. Yours is most likely different.

**Comment by enoch85** [comment]
Sorry, I forgot one file in the mix. Please check the new instructions here: https://github.com/ltsp/community/wiki/Single-guest-session-over-NFS#prepare-the-nfs-mount

**Comment by enoch85** [comment]
@alkisg Upgraded to 19.11 and it stopped working. Any idea what could have caused that?

**Comment by alkisg** [comment]
After checking via IRC/VNC :)

I don't think it was related to the update, but it seems to work fine now, so I don't know what the temporary problem was.

**Comment by enoch85** [comment]
Wiki updated now, I think this can be closed.

**Comment by wolfclan68** [comment]
Is there something wrong with the line below from your ltsp.conf? When it is in my conf I get this error /usr/sbin/ltsp: 106: eval: Syntax error: Unterminated quoted string. If I remove it, it runs, but I get a boot up like a new user account.

sed 's|^Exec=|&/etc/ltsp/ltsp-session |' -i /usr/share/xsessions/*.desktop
"

**Comment by enoch85** [comment]
I just did a fresh install today, following the guide, and it worked out of the box. That's all I can say.

I'm not exactly sure what that specific command does, but I think it runs the rsync script.

**Comment by wolfclan68** [comment]
Ran through everything again and it worked this time. Thank you!
